src/Fantomas.Tests/ClassTests.fs

```diff
@@ -896,3 +896,38 @@ type MaybeBuilder() =
         ) : 'U option =
         Option.bind binder value
 """
+
+[<Test>]
+let ``avoid vanity alignment when calling base constructor`` () =
```

Comment:
Please add the issue number that the end of the test. See [Contribution guidelines](https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#guidelines).